src/pretix/control/templates/pretixcontrol/organizers/detail.html

```diff
@@ -7,7 +7,7 @@
         {% blocktrans with name=organizer.name %}Organizer: {{ name }}{% endblocktrans %}
         <a href="{% url "control:organizer.edit" organizer=organizer.slug %}"
            class="btn btn-default">
-            <span class="fa fa-edit"></span>
+            <span class="fa fa-sliders"></span>
```

Comment:
May I ask why you changed this? We use ``fa-edit`` a lot for editing (and ``fa-wrench`` for event configuration) and I think it is better to stay consistent than to introduce ``fa-sliders``.